### PR TITLE
Update Blockchain -> what-does-ethereum-do-differently.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Split insight into 2 smaller ones

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/blockchain/fundamentals/ethereum-i/what-does-ethereum-do-differently.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/blockchain/fundamentals/ethereum-i/what-does-ethereum-do-differently.md?ref=b3f986ed3d2890d25862386a1b8da77e8bbd1cc9)

File: blockchain/fundamentals/ethereum-i/what-does-ethereum-do-differently-ii.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  - introduction
+
+  - deep
+
+---
+# What Does Ethereum Do Differently? (Part 2)
+
+---
+## Content
+
```
Comment (kapnobatai137) [inline blockchain/fundamentals/ethereum-i/what-does-ethereum-do-differently-ii.md]: how does this introduction sound? trying to make this insight more individual

cc @nem035
```suggestion
      	       
Ethereum couldn't be more different than Bitcoin. From using an account model, and having special transactions to solidity and token stardards. In this insight we'll take a look at some of those differences.

```

Comment (nem035) [reply blockchain/fundamentals/ethereum-i/what-does-ethereum-do-differently-ii.md]: I think "couldn't be more different", is a bit of an overstatement. I'd say Ethereum couldn't be more different than a pretzel or something but not Bitcoin 😆 

Anyhow, how about this?

```suggestion
      	       
Ethereum is different than Bitcoin in many ways. It has an account model, special transactions, a programming language (called Solidity) and token standards. In this insight we'll take a look at some of those differences.

```

Comment (kapnobatai137) [reply blockchain/fundamentals/ethereum-i/what-does-ethereum-do-differently-ii.md]: yeah, I went kinda hard when I said "couldn't be more different" :rofl:

**Pull request merged**